gapic-generator-java/src/main/java/com/google/api/generator/gapic/composer/comment/SettingsCommentComposer.java

```diff
@@ -120,22 +120,22 @@ public CommentStatement getTransportProviderBuilderMethodComment() {
   }
 
   public static CommentStatement createCallSettingsGetterComment(
-      String javaMethodName, boolean isMethodDeprecated) {
-    String methodComment = String.format(CALL_SETTINGS_METHOD_DOC_PATTERN, javaMethodName);
-    return isMethodDeprecated
-        ? toDeprecatedSimpleComment(methodComment)
-        : toSimpleComment(methodComment);
+      String javaMethodName, boolean isMethodDeprecated, boolean isMethodInternal) {
+    return toDeprecatedInternalSimpleComment(
+        String.format(CALL_SETTINGS_METHOD_DOC_PATTERN, javaMethodName),
+        isMethodDeprecated,
+        isMethodInternal);
   }
 
   public static CommentStatement createBuilderClassComment(String outerClassName) {
     return toSimpleComment(String.format(BUILDER_CLASS_DOC_PATTERN, outerClassName));
   }
 
   public static CommentStatement createCallSettingsBuilderGetterComment(
-      String javaMethodName, boolean isMethodDeprecated) {
+      String javaMethodName, boolean isMethodDeprecated, boolean isMethodInternal) {
     String methodComment = String.format(CALL_SETTINGS_BUILDER_METHOD_DOC_PATTERN, javaMethodName);
-    return isMethodDeprecated
-        ? toDeprecatedSimpleComment(methodComment)
+    return isMethodDeprecated || isMethodInternal
```

Comment:
iiuc, `toDeprecatedInternalSimpleComment()` can also handle simple comment case, this conditional operator is redundant.
Also, perhaps rename to clarify and remove toSimpleComment() if not used anymore.